src/components/Clock.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  clockTime: Date,
+};
+
+const timeNormalize = (date: Date) => date.toUTCString().slice(-12, -4);
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    clockTime: new Date(),
+  };
+
+  timeIntervalId = 0;
```

Comment:
I guess it would be more clear if it were `timeTimerId` but this one is up to you